apps/src/Sound.js

```diff
@@ -339,22 +340,55 @@ Sound.prototype.getPlayableFile = function() {
   return false;
 };
 
+Sound.prototype.getPlayableBytes = function() {
+  try {
+    if (!window.Audio) {
+      return false;
+    }
+
+    var audioTest = new window.Audio();
+    if (
+      this.config.hasOwnProperty('bytes') &&
+      audioTest.canPlayType('audio/wav')
+    ) {
+      return this.config.bytes;
+    }
+  } catch (e) {}
+
+  return false;
+};
+
 Sound.prototype.preload = function() {
   var file = this.getPlayableFile();
-  if (!file) {
+  var bytes = this.getPlayableBytes();
+  if (!file && !bytes) {
     return;
   }
 
   if (!this.config.forceHTML5 && window.AudioContext && this.audioContext) {
     var self = this;
-    this.preloadViaWebAudio(file, function(buffer) {
-      self.reusableBuffer = buffer;
-    });
+    if (file) {
+      this.preloadViaWebAudio(file, function(buffer) {
+        self.reusableBuffer = buffer;
+      });
+    } else {
+      self.audioContext.decodeAudioData(bytes, function(buffer) {
+        self.reusableBuffer = buffer;
+        self.onSoundLoaded();
+      });
+    }
     return;
   }
 
   if (window.Audio) {
     var audioElement = new window.Audio(file);
```

Comment:
What's the purpose of having this line, and the one below after you check `if (file)` ?

Reply:
oops, I meant to remove that